src/vs/workbench/browser/parts/editor/editorGroupView.ts

```diff
@@ -1015,55 +1021,11 @@ export class EditorGroupView extends Themable implements IEditorGroupView {
 		return this.model.isActive(editor);
 	}
 
-	async selectEditor(editor: EditorInput, active?: boolean): Promise<void> {
-		if (active) {
-			await this.doOpenEditor(editor, { activation: EditorActivation.ACTIVATE }, { selected: true });
+	async setSelection(activeSelectedEditor: EditorInput, inactiveSelectedEditors: EditorInput[]): Promise<void> {
+		if (!this.model.isActive(activeSelectedEditor)) {
+			await this.doOpenEditor(activeSelectedEditor, { activation: EditorActivation.ACTIVATE }, { inactiveSelectedEditors });
```

Comment:
I suggest to remove the need for `inactiveSelectedEditors` and simply call `this.model.setSelection` always below.

Reply:
I'll give it a try, not sure if everything will come together correctly.

Reply:
So the problem now is that there is 2 title updates going on. One with only the active editor and the second with the selection. This causes an ugly flicker